dev-python/matplotlib-venn/matplotlib-venn-0.11.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Routines for plotting area-weighted two- and three-circle venn diagrams"
+HOMEPAGE="http://pysurfer.github.com"
+SRC_URI="'mirror://pypi/m/${PN}/${P}.zip"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+RDEPEND="
+	dev-python/matplotlib[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]"
+DEPEND="test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
+
+python_test() {
+	distutils_install_for_testing
+	cd "${TEST_DIR}"/lib || die
+	pytest -v || die "Tests fail with ${EPYTHON}"
```

Comment:
Have you executed the test phase with `ebuild matplotlib-venn-0.11 test`? Are the tests passing? I don't think so, since I assume it should be `py.test` instead of `pytest`.